autoload/coverage.vim

```diff
@@ -27,6 +27,7 @@ endif
 if !exists('s:coverage_states')
   let s:coverage_states = ['covered', 'uncovered', 'partial']
 endif
+let s:highlights_defined = 0
```

Comment:
Don't you need to guard this with `if exists('s:highlights_defined')`? autoload files can be re-executed whenever and AFAICT this shouldn't be reset to 0 in that case.